packages/timeline-state-resolver/src/integrations/sofieChef/index.ts

```diff
@@ -145,15 +150,40 @@ export class SofieChefDevice extends DeviceWithState<SofieChefState, DeviceOptio
 			delete this.reconnectTimeout
 
 			this._setupWSConnection()
-				.then(() => {
+				.then(async () => {
 					// is connected, yay!
+
+					// Resync state:
+					await this.resyncState()
 				})
 				.catch(() => {
 					// Unable to reconnect, try again later:
 					this.tryReconnect()
 				})
 		}, RECONNECT_WAIT_TIME)
 	}
+	async resyncState() {
```

Comment:
I don't think this method needs to be public? If it needs to be accessed outside of the device class perhaps adding a tsr action would be better.